PVTPackage/source/MultiphaseSystem/MultiphaseSystemProperties/DeadOilFlashMultiphaseSystemProperties.cpp

```diff
@@ -17,26 +17,55 @@
 #include "pvt/pvt.hpp"
 
 #include <cmath>
+#include <algorithm>
 
 namespace PVTPackage
 {
 
-DeadOilFlashMultiphaseSystemProperties::DeadOilFlashMultiphaseSystemProperties( std::size_t nComponents )
+DeadOilFlashMultiphaseSystemProperties::DeadOilFlashMultiphaseSystemProperties( std::vector< pvt::PHASE_TYPE > const & phases )
   :
-  BlackOilDeadOilMultiphaseSystemProperties( nComponents )
+  BlackOilDeadOilMultiphaseSystemProperties( phases )
 {
+  size_t const nComponents = phases.size();
```

Comment:
Same here.

Reply:
Done with the new getter suggested below.